CMakeLists.txt

```diff
@@ -1,13 +1,14 @@
 cmake_minimum_required(VERSION 3.0)
+cmake_policy(SET CMP0075 NEW)
```

Comment:
Could you try unsetting the CMP0075 policy and setting the cmake minimum to 3.12. I don't understand this cmake policy stuff well and I'm curious to see if it works based on this conversation: https://reviews.llvm.org/D82074

Reply:
It works :-)
I also changed it in VCalcBase